torch_em/data/datasets/light_microscopy/cvz_fluo.py

```diff
@@ -0,0 +1,167 @@
+"""The CVZ-Fluo dataset contains annotations for cell and nuclei segmentation in
+fluorescence microscopy images.
+
+NOTE: You need to install 'synapseclient`, create an account and authentication token on Synapse,
+and follow the documentation to create your authentication config: https://python-docs.synapse.org/tutorials/authentication/
+and install using 'synapse get -r syn27624812'.
+
+The dataset is from the publication https://doi.org/10.1038/s41597-023-02108-z.
+Please cite it if you use this dataset for your research.
+"""  # noqa
+
+import os
+from glob import glob
+from tqdm import tqdm
+from pathlib import Path
+from natsort import natsorted
+from typing import Union, Literal, Tuple, Optional, List
+
+import imageio.v3 as imageio
+from skimage.measure import label as connected_components
+
+from torch.utils.data import Dataset, DataLoader
+
+import torch_em
+
+from .. import util
+from .neurips_cell_seg import to_rgb
+
+
+URL = "https://www.synapse.org/Synapse:syn27624812/"
+
+
+def get_cvz_fluo_data(path: Union[os.PathLike, str], download: bool = False):
+    """Obtain the CVZ-Fluo dataset.
+
+    Args:
+        path: Filepath to a folder where the downloaded data is saved.
+        download: Whether to download the data if it is not present.
+    """
+    if download:
+        raise NotImplementedError("Automatic download is not implemented for this dataset.")
+
+    data_dir = os.path.join(path, r"Annotation Panel Table.xlsx")
```

Comment:
Why do you need to cast this to a raw string here?

Reply:
With filenames having space between them, they create some issues on Windows. Scripting the paths with spaces as raw string usually avoids this issue!